central/sensor/service/pipeline/clustermetrics/pipeline.go

```diff
@@ -68,6 +69,13 @@ func (p *pipelineImpl) Run(
 	clusterMetrics := msg.GetClusterMetrics()
 	p.metricsStore.Set(clusterID, clusterMetrics)
 	p.telemetryMetrics.SetClusterMetrics(clusterID, clusterMetrics)
+	if err := p.usageStore.UpdateUsage(ctx, clusterID, &storage.SecuredUnits{
+		NumNodes:    clusterMetrics.GetNodeCount(),
+		NumCpuUnits: clusterMetrics.GetCpuCapacity(),
+	}); err != nil {
+		logging.GetRateLimitedLogger().Warn(
+			"Error while trying to update secured units usage:", err.Error())
```

Comment:
```suggestion
			"Error while trying to update secured units product usage: ", err.Error())
```

Reply:
Secured units usage is well precise here.